### PR TITLE
exa, eza: add Polish translation

### PR DESCRIPTION
**Pull request by TheBulaDev**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.pl/common/exa.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Wyświetl listę plików, po jednym w linii:
+
+`exa --oneline`
+
+- Wyświetl wszystkie pliki, łącznie z ukrytymi:
+
+`exa --all`
+
+- Wyświetl listę wszysktich plików ze szczegółami (uprawnienia, właściciel, wielkość i data zmiany):
```
Comment (acuteenvy) [inline pages.pl/common/exa.md:14]:
```suggestion
- Wyświetl listę wszystkich plików ze szczegółami (uprawnienia, właściciel, wielkość i data zmiany):
```

File: pages.pl/common/exa.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`exa --long --all`
+
+- Wyświetl listę plików, posortowane względem wielkości pliku, od największego:
```
Comment (acuteenvy) [inline pages.pl/common/exa.md:18]:
```suggestion
- Wyświetl listę plików posortowaną względem wielkości pliku, od największego:
```

File: pages.pl/common/exa.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`exa --reverse --sort={{size}}`
+
+- Wyświetl drzewko plików, trzy poziomy:
```
Comment (acuteenvy) [inline pages.pl/common/exa.md:22]:
```suggestion
- Wyświetl drzewko plików (trzy poziomy):
```

File: pages.pl/common/exa.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`exa --long --tree --level={{3}}`
+
+- Wyświetl listę plików, posorotwane wzgledem daty zmiany, od najstarszego:
```
Comment (acuteenvy) [inline pages.pl/common/exa.md:26]:
```suggestion
- Wyświetl listę plików posortowaną wzgledem daty zmiany, od najstarszego:
```

File: pages.pl/common/exa.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`exa --long --sort={{modified}}`
+
+- Wyświetl listę plików wraz z nagłówkiem, ikoną i statustem Git:
```
Comment (acuteenvy) [inline pages.pl/common/exa.md:30]:
```suggestion
- Wyświetl listę plików wraz z nagłówkiem, ikoną i statusem Git:
```

File: pages.pl/common/exa.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`exa --long --header --icons --git`
+
+- Wyświetl listę plików, ignoruj pliki z `.gitignore`:
```
Comment (acuteenvy) [inline pages.pl/common/exa.md:34]:
```suggestion
- Wyświetl listę plików, ignorując pliki z `.gitignore`:
```

Comment (TheBulaDev) [reply pages.pl/common/exa.md:34]: Dziękuje za wszystkie sugestie

**Comment by acuteenvy** [comment]
I created the translation for `eza` since it has the exact same examples.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**